Dockerfile

```diff
@@ -54,6 +54,10 @@ COPY /strategy_plugins/*            /usr/share/ansible/plugins/strategy/
 COPY /modules/*                     /usr/share/ansible/plugins/modules/
 RUN mv /opt/cray/ansible/modules/*    /usr/share/ansible/plugins/modules/
 
+# Stage ARA plugins
+RUN mkdir -p /usr/share/ansible/plugins/ara/
+RUN cp $(python3 -m ara.setup.callback_plugins)/*.py /usr/share/ansible/plugins/ara/
```

Comment:
Yes.  This is how they recommend getting the path of the plugin.  I'm just copying it to a consistent location so that we can more easily document it.